package.json

```diff
@@ -26,7 +26,7 @@
     "whatwg-fetch": "^2.0.3"
   },
   "peerDependencies": {
```

Comment:
We will need to remove the `peerDependencies` since we don't ship `explorer-ui-server` as npm package anymore with related changes. This will confuse `npm`.

Reply:
true, i agree

Reply:
removed peer dependency